src/xwindows/keyboard.rs

```diff
@@ -0,0 +1,590 @@
+         // xkb::KEY_XF86Pictures =>             KeyCode::Pictures,
+         // xkb::KEY_XF86Music =>                KeyCode::Music,
+         // xkb::KEY_XF86Battery =>              KeyCode::Battery,
+         // xkb::KEY_XF86Bluetooth =>            KeyCode::Bluetooth,
+         // xkb::KEY_XF86WLAN =>                 KeyCode::WLAN,
+         // xkb::KEY_XF86UWB =>                  KeyCode::UWB,
+         // xkb::KEY_XF86AudioForward =>         KeyCode::AudioForward,
+         // xkb::KEY_XF86AudioRepeat =>          KeyCode::AudioRepeat,
+         // xkb::KEY_XF86AudioRandomPlay =>      KeyCode::AudioRandomPlay,
+         // xkb::KEY_XF86Subtitle =>             KeyCode::Subtitle,
+         // xkb::KEY_XF86AudioCycleTrack =>      KeyCode::AudioCycleTrack,
+         // xkb::KEY_XF86Time =>                 KeyCode::Time,
+         // xkb::KEY_XF86Select =>               KeyCode::Select,
+         // xkb::KEY_XF86View =>                 KeyCode::View,
+         // xkb::KEY_XF86TopMenu =>              KeyCode::TopMenu,
+         // xkb::KEY_XF86Red =>                  KeyCode::Red,
+         // xkb::KEY_XF86Green =>                KeyCode::Green,
+         // xkb::KEY_XF86Yellow =>               KeyCode::Yellow,
+         // xkb::KEY_XF86Blue =>                 KeyCode::Blue,
+         // xkb::KEY_XF86Bluetooth =>            KeyCode::Bluetooth,
+         // xkb::KEY_XF86Suspend =>              KeyCode::Suspend,
+         // xkb::KEY_XF86Hibernate =>            KeyCode::Hibernate,
+         // xkb::KEY_XF86TouchpadToggle =>       KeyCode::TouchpadToggle,
+         // xkb::KEY_XF86TouchpadOn =>           KeyCode::TouchpadOn,
+         // xkb::KEY_XF86TouchpadOff =>          KeyCode::TouchpadOff,
+         // xkb::KEY_XF86AudioMicMute =>         KeyCode::MicMute,
+         // xkb::KEY_XF86Launch0 =>              KeyCode::Launch0, // ### Qt 6: remap properly
+         // xkb::KEY_XF86Launch1 =>              KeyCode::Launch1,
+         // xkb::KEY_XF86Launch2 =>              KeyCode::Launch2,
+         // xkb::KEY_XF86Launch3 =>              KeyCode::Launch3,
+         // xkb::KEY_XF86Launch4 =>              KeyCode::Launch4,
+         // xkb::KEY_XF86Launch5 =>              KeyCode::Launch5,
+         // xkb::KEY_XF86Launch6 =>              KeyCode::Launch6,
+         // xkb::KEY_XF86Launch7 =>              KeyCode::Launch7,
+         // xkb::KEY_XF86Launch8 =>              KeyCode::Launch8,
+         // xkb::KEY_XF86Launch9 =>              KeyCode::Launch9,
+         // xkb::KEY_XF86LaunchA =>              KeyCode::LaunchA,
+         // xkb::KEY_XF86LaunchB =>              KeyCode::LaunchB,
+         // xkb::KEY_XF86LaunchC =>              KeyCode::LaunchC,
+         // xkb::KEY_XF86LaunchD =>              KeyCode::LaunchD,
+         // xkb::KEY_XF86LaunchE =>              KeyCode::LaunchE,
+         // xkb::KEY_XF86LaunchF =>              KeyCode::LaunchF,
+            _ => {return None;}
+        };
+    Some(res)
+}
```

Comment:
this is a really long function (especially with all the commented out bits), so it would be good to move it into its own file (perhaps back into `xkeysyms.rs`?)